make-release.sh

```diff
@@ -120,7 +120,10 @@ docker build -t "${PROJECT_CLONE_QUAY_REPO}" -f ./project-clone/Dockerfile .
 docker push "${PROJECT_CLONE_QUAY_REPO}"
 
 set -x
-bash -x ./deploy/generate-deployment.sh --use-defaults --default-image ${QUAY_REPO}
+bash -x ./deploy/generate-deployment.sh \
+  --use-defaults \
+  --default-image ${QUAY_REPO} \
+  --project-clone-image ${PROJECT_CLONE_QUAY_REPO}
```

Comment:
Well, it's out of the current PR scope, I must miss it in the previous PRs, and I think it's just sync with `QUAY_REPO` but it's not actually a good name. It's not just repo but image
```suggestion
  --project-clone-image ${PROJECT_CLONE_QUAY_IMG}
```

Reply:
Please consider changing QUAY_REPO -> DWO_IMG or QUAY_DWO_IMG